src/sagemaker/predictor.py

```diff
@@ -134,15 +135,14 @@ def delete_model(self):
         for model_name in self._model_names:
             self.sagemaker_session.delete_model(model_name)
 
-    def _get_endpoint_config_desc(self):
+    def _get_endpoint_config_name(self):
         endpoint_desc = self.sagemaker_session.sagemaker_client.describe_endpoint(EndpointName=self.endpoint)
-        self._endpoint_config_name = endpoint_desc['EndpointConfigName']
+        endpoint_config_name = endpoint_desc['EndpointConfigName']
+        return endpoint_config_name
+
+    def _endpoint_config_desc_and_model_names(self):
```

Comment:
This method used to set the _endpoint_config_name and now it doesn't.

I think it is okay to rename this back lol

Reply:
Changed back.